api/services/locationService.ts

```diff
@@ -33,8 +36,17 @@ function mapToLocationDto(agency: ScosAgencyDto): LocationDto | null {
   // TODO: find out how we can determine if locations have handicap access
   const handicapAccessFlag = 'N';
 
+  // retrieve FreshTrak data if sub_category is 'Emergency Food'
+  var freshtrakData: freshtrakLocationDto | null;
```

Comment:
I'm surprised our linter is not catching this - are you running the linter?
```suggestion
  let freshtrakData: freshtrakLocationDto | null;
```

Reply:
No.  I need to look into how to run linter locally.  I have been relying on VS code for warnings.

Reply:
VS Code should be able to automatically run it every time you save files, otherwise you should be able to run `npm run lint:fix` to do the whole project